index.html

```diff
@@ -259,6 +259,25 @@ <h1>Important Terms</h1>
 			  <p>Operating systems and other platforms provide a set of interfaces that expose information about <a class="termref">objects</a> and <a class="termref">events</a> to <a>assistive technologies</a>. Assistive technologies use these interfaces to get information about and interact with those <a class="termref">widgets</a>. Examples of accessibility APIs are <a href="https://docs.microsoft.com/en-us/windows/win32/winauto/microsoft-active-accessibility">Microsoft Active Accessibility</a> [[MSAA]], <a href="https://docs.microsoft.com/en-us/windows/win32/winauto/entry-uiauto-win32">Microsoft User Interface Automation</a> [[UI-AUTOMATION]], <abbr title="Microsoft Active Accessibility">MSAA</abbr> with <cite><a href="https://docs.microsoft.com/en-us/windows/win32/winauto/iaccessibleex"><abbr title="User Interface Automation">UIA</abbr> Express</a></cite> [[UIA-EXPRESS]], the
 				  <a href="https://developer.apple.com/documentation/appkit/nsaccessibility">Mac <abbr title="OS Ten">OS X</abbr> Accessibility Protocol</a> [[AXAPI]], the <cite><a href="https://developer.gnome.org/atk/unstable/">Linux/Unix Accessibility Toolkit</a></cite> [[ATK]] and <cite><a href="https://developer.gnome.org/libatspi/stable/">Assistive Technology Service Provider Interface</a></cite> [[AT-SPI]], and <a href="https://wiki.linuxfoundation.org/accessibility/iaccessible2/start">IAccessible2</a> [[IAccessible2]].</p>
 			</dd>
+			<dt><dfn data-export="" data-lt="accessibility child|accessibility children|child element|child|children|child elements">Accessibility child</dfn></dt>
+			<dd>
+				<p>To calculate the <a>accessibility children</a> of an element `X`, first considered a modified <abbr title="Document Object Model">DOM</abbr> tree where elements (and their subtrees) specified by `aria-owns` are reparented to the element with `aria-owns` set.
+				<p>Now, an <a>accessibility child</a> of an <a>element</a> `X` is considered to be any of the following:
+				<ul>
+					<li>A DOM child of `X` in that modified DOM tree.</li>
+					<li>A DOM descendant of `X` in that modified DOM tree where the only intervening elements between the descendant and `X` have role <rref>generic</rref> or <rref>none</rref>.</li>
+				</ul>
+				</p>
+                                <p class="note">Accessibility child, accessibility descendant and accessibility parents are terms used to defined required relationships between different DOM elements with ARIA roles or properties. The terms do not refer to a platform accessibility API trees and the relationship between platform accessiblity nodes.</p>
```

Comment:
should these _always_ be terms used to defined required relationships?  I'd very much like to use these terms (and had started to in a draft pr) to communicate relationships for scoping.  I guess the way i'm looking at it there is that the relationships are not _required_ but in the situations where applicable, being a child of X would give the child element Y a contextual role, or not.

Reply:
Good point. I'll revise this.